### ISSUE TITLE
11.3.0-36E Safehax error

### ISSUE THREAD

**Issue by NightDante** [TiniVi/safehax#21]
Hi, I'm on 11.3.0-36E, and safehax didn't work for me. I tried everything. Read a lot of forums, and nothing. I tried Arc13's safehaxes, but it's not working neither. Homebrew and USDploit are working, but Safehax is not. Can someone help me?

**Comment by Masamune3210** [comment]
Safehax was patched in 11.3, so it will never work.

**Comment by TiniVi** [comment]
It was patched on 11.4, don't spread misinformation.

**Comment by TiniVi** [comment]
Can you share which error is occuring? A blackscreen or misc?

**Comment by NightDante** [comment]
I got blackscreen after my 3ds turnoff.  I tried everything. Format my sdcard to fat32 and and cluster size 32kb. I tried some safehax version. USDploit is working, but safehax never run. I always wait 30sec or 1min. But nothing.

**Comment by NightDante** [comment]
![3dsfa](https://cloud.githubusercontent.com/assets/27772267/25230453/409bf3ac-25d4-11e7-9fa4-2d914d06137e.png)
Here is my root, maybe it's help.

**Comment by Masamune3210** [comment]
Sorry, was unaware of there being a fix to Big N patching it in 11.3

**Comment by TiniVi** [comment]
I don't really see the issue here, but I can give you a debug build that may help (?) in a few hours

**Comment by NightDante** [comment]
I see, and thank you.

**Comment by ghost** [comment]
I also have that problem, it gets a blackscreen or a "Pm init failed!", version 11.3.0 - 36E

**Comment by NightDante** [comment]
Try your sd card format to fat32 and cluster size 32kb. Maybe it's help your problem.

**Comment by ghost** [comment]
I formated it and reinstalled, seems to work! Now I have to format again
sinsce avast fucked up one of the files -_-

2017-04-20 21:34 GMT+01:00 NightDante <notifications@github.com>:

**Comment by NightDante** [comment]
Something news?

**Comment by ghost** [comment]
it works perfectly now ^^ So thanks for the interest!

2017-04-23 12:33 GMT+01:00 NightDante <notifications@github.com>:

**Comment by NightDante** [comment]
What was your problem? Pls tell me.

**Comment by ghost** [comment]
Avast scaned the file (And somehow) made it impossible to read for the 3ds
so then the 3ds gave an error since the file was broken.


2017-04-25 14:19 GMT+01:00 NightDante <notifications@github.com>:

**Comment by NightDante** [comment]
I'm not using Avast, so it not good news for me. Any idea?

**Comment by ghost** [comment]
yeah, just make sure that when you put the file onto the SD the size is the
same and that it has an icon when 3ds shows you the homebrew ^^

2017-04-25 22:07 GMT+01:00 NightDante <notifications@github.com>:

**Comment by NightDante** [comment]
The icon is just white icon, and file size is same.

**Comment by ghost** [comment]
weird, in my case it was a different file size and a ? Icon, I dont know a
lot about how to fix problems so I can only say the classic "Try formatting
and do it again" :( sorry

2017-04-26 13:15 GMT+01:00 NightDante <notifications@github.com>:

**Comment by chaudhrysahil123** [comment]
Hello, I am on n3ds xl 11.3.0.36E and i am getting black screen every time i use safehax. udsploit and homebrew are working fine. I have also formatted my sd card to fat32 and 32kb.

**Comment by wezajexa** [comment]
I'm in the same boat as chaudhrysahil123: on a n3ds 11.3.0.36U and followed `3ds.guide` as precisely as I could. Soundhax, udsploit, homebrew all work fine, but I get stuck on black screen whenever trying safehax.

Tried reformatting to fat32/32kb, with & without game card, tried repeatedly, same result.

**Comment by stenocyb** [comment]
I have exactly the same issue on 11.3.0-36E

**Comment by stenocyb** [comment]
Nvm I fixed it by jumping directly to the "Finalizing setup" step on 3ds.guide. Apparently safehax was already installed somehow.

**Comment by NightDante** [comment]
Which safehax do you use? r21 or r19?

**Comment by TiniVi** [comment]
Try the latest commit (r25).

**Comment by red09** [comment]
hey, I'm having the same issue as Dante. I've also tried Arc13's build and I used r27. My system is also the same as his, but it's U.
